tests/hw5/test_task1.py

```diff
@@ -0,0 +1,47 @@
+from datetime import timedelta
+
+from hw5.oop_1 import Homework, Student, Teacher
+
+
+def test_summoning_student():
+    student = Student("Roman", "Petrov")
+
+    assert student.first_name == "Roman"
+    assert student.last_name == "Petrov"
+
+
+def test_summoning_teacher():
+    teacher = Teacher("Daniil", "Shadrin")
```

Comment:
this object is being created multiple times per test module. can we simplify it?